package.json

```diff
@@ -20,10 +20,10 @@
   "devDependencies": {
     "chai": "^4.3.6",
     "mocha": "^10.0.0",
-    "mongodb-memory-server": "^8.5.2"
+    "mongodb-memory-server": "^10.1.2"
```

Comment:
This bumps the in-memory mongodb version from 5 to 7 ([ref](https://typegoose.github.io/mongodb-memory-server/docs/guides/mongodb-server-versions#mongodb-memory-server-core-version-table))